test/support/move_only.h

```diff
@@ -0,0 +1,52 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _MOVE_ONLY_H
+#define _MOVE_ONLY_H
+
+#include <utility>
```

Comment:
Since `swap` was removed, this header is not necessary any more.

Reply:
No: class `MoveOnly` still used in tests

Reply:
The comment is about `<utility>` header inclusion, not the entire `move_only.h`